third_party/silabs/SiWx917_sdk.gni

```diff
@@ -169,7 +241,7 @@ template("siwx917_sdk") {
       defines += [ "DIC_ENABLE=1" ]
     }
 
-    libs += [ "${sdk_support_root}/platform/emdrv/nvm3/lib/libnvm3_CM4_gcc.a" ]
+    libs += [ "${efr32_sdk_root}/platform/emdrv/nvm3/lib/libnvm3_CM4_gcc.a" ]
```

Comment:
This seems wrong. Libs are committed to the sdk_support not the efr32_sdk_root....

Reply:
We are trying to use the library present in gecko SDK
as that will be latest.

the one present in sdk_support is copied from gecko sdk only

Reply:
This will make CI fail on github. that was I asked for it to be reverted.